multimedia/mythtv.27/Portfile

```diff
@@ -10,10 +10,10 @@ set shorthash       3c7582e9
 set majorversion    .27
 set minorversion    .7
 set lastcommit      20180201
-set corerev         4
-set pluginsrev      3
+set corerev         5
+set pluginsrev      4
 github.setup        MythTV mythtv ${shorthash}
-revision            0
+revision            1
```

Comment:
I'm not sure about all these revisions.

Reply:
Yeah, I don't think the metaport revision needs to be bumped. See what I did when updating mythtv.28.